lib/engine/game/g_1825/step/track_and_token.rb

```diff
@@ -74,6 +74,10 @@ def lay_tile_action(action, entity: nil, spender: nil)
             if tile_lay[:cannot_reuse_same_hex] && @round.laid_hexes.include?(action.hex)
               raise GameError, "#{action.hex.id} cannot be layed as this hex was already layed on this turn"
             end
+            if (tile.name == '888' || tile.name == '887') &&
+               action.hex.id == 'Q13' && (entity.name != 'NS')
```

Comment:
Inconsistent use of parens

Reply:
Oh bother. Rubocop likes to make you take them out, whereas I like to put them in. I think that's how this happened. Correcting.